aiida/storage/psql_dos/orm/groups.py

```diff
@@ -249,7 +270,7 @@ def check_node(node):
 
         list_nodes = []
 
-        with utils.disable_expire_on_commit(self.backend.get_session()) as session:
```

Comment:
Same as above, why does this no longer need to be disabled?

Also note, this removes the only two uses of the `disable_expire_on_commit` function

Reply:
restored the change